src/test/ui/consts/issue-66787.rs

```diff
@@ -0,0 +1,39 @@
+// build-pass
+// compile-flags: --crate-type lib
+
+// Regression test for ICE which occurred when const propagating an enum whose discriminant
+// niche triggered an integer underflow conmupting a delta.
```

Comment:
Is there a way to also trigger this ICE in standalone Miri? I'd prefer to have such important parts of the Miri engine not just covered indirectly through an optimization.

Reply:
Given https://github.com/rust-lang/rust/pull/66857#issuecomment-559797758 I'm not sure that's possible, since this is necessarily an initialization of an uninhabited variant, and that's dead code.

Miri likely wouldn't have had this bug for so long if it was actually executing the code that is being constant-folded here.